app/javascript/components/UserList/index.js

```diff
@@ -14,7 +14,6 @@ const UserList = ({ users, destroySignup }) => (
             userId={user.id}
             image={user.photo}
             name={user.name}
-            subtitle={user.group}
```

Comment:
I did some digging and it turns out user does have a field called `group`, though not all users have that field populated.

```ruby
irb(main):007:0> User.where(group: nil).count
=> 1206
irb(main):008:0> User.where.not(group: nil).count
=> 1712
```

Reply:
That's interesting. What is an example of a `group`?

We could either add the meta data to the subtitle section by fixing the graphQL `UserEntry` query or leave it as is (removing the subtitle section, as it's empty anyway).

It may as well be too much clutter if the group information isn't interesting or relevant anyway.

Thoughts?

Reply:
> What is an example of a group?

"Growth Marketing"

> We could either add the meta data to the subtitle section by fixing the graphQL UserEntry query or leave it as is (removing the subtitle section, as it's empty anyway).

I reckon we should fix the GraphQL query and populate `group`, since more than half of users have `group` field populated.

Reply:
@xuebingli fixed!